extensions/security/deployment/src/main/java/io/quarkus/security/deployment/SecurityProcessor.java

```diff
@@ -251,10 +267,9 @@ public void write(String s, byte[] bytes) {
                 }
             }, "io.quarkus.security.BouncyCastleFeature", null, Object.class.getName(),
                     org.graalvm.nativeimage.hosted.Feature.class.getName());
-            file.addAnnotation("com.oracle.svm.core.annotate.AutomaticFeature");
 
-            MethodCreator beforeAn = file.getMethodCreator("beforeAnalysis", "V",
-                    org.graalvm.nativeimage.hosted.Feature.BeforeAnalysisAccess.class.getName());
+            MethodCreator beforeAn = file.getMethodCreator("afterRegistration", "V",
+                    org.graalvm.nativeimage.hosted.Feature.AfterRegistrationAccess.class.getName());
```

Comment:
Is this change intentional?

Reply:
Yes. Apparently without this change the BC provider doesn't get registered on time. I assume this has to do with the ordering that the different Features are being ran which is impacted by removing the `AutomaticFeature` annotation from our features. Furthermore, it should have been that way anyway, see https://github.com/oracle/graal/issues/2800#issuecomment-702480444.

Reply:
Can you also rename the variable to reflect thia change?